src/components/ProactiveAppReviewModalManager.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {useCallback} from 'react';
+import useOnyx from '@hooks/useOnyx';
+import useProactiveAppReview from '@hooks/useProactiveAppReview';
+import requestStoreReview from '@libs/actions/StoreReview';
+import {respondToProactiveAppReview} from '@libs/actions/User';
+import Navigation from '@libs/Navigation/Navigation';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type {AppReviewResponse} from '@src/types/onyx/AppReview';
+import ProactiveAppReviewModal from './ProactiveAppReviewModal';
+
+const CONCIERGE_POSITIVE_MESSAGE = "Hi there! I'm glad to hear you're enjoying Expensify. What's your favorite thing about the app? Thanks!";
+const CONCIERGE_NEGATIVE_MESSAGE = "Hi there! I'm sorry to hear you aren't fully satisfied with Expensify. What's your #1 frustration? Thanks!";
+
+function ProactiveAppReviewModalManager() {
+    const {shouldShowModal, proactiveAppReview} = useProactiveAppReview();
+    const [conciergeReportID] = useOnyx(ONYXKEYS.CONCIERGE_REPORT_ID, {canBeMissing: true});
+
+    const handleResponse = useCallback(
+        (response: AppReviewResponse, message?: string) => {
+            // Call the action which will create optimistic comment (if message provided) and call API
```

Comment:
```suggestion
            // Call the action which will create optimistic comment (if message provided) and call API
```
            // Call the action which will create optimistic comment (if message is provided) and call API

Reply:
I squinted but cant see what is changing 😄

Reply:
😄 updated